packages/plugin-global-user-properties/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@amplitude/plugin-global-user-properties",
+  "version": "2.1.0",
+  "description": "",
```

Comment:
Any value in adding desc here?

Reply:
oh good catch, I copy-pasta'd this from another plugin and should add this